cmd/kanikoExecute_test.go

```diff
@@ -43,6 +43,18 @@ type kanikoFileMock struct {
 	fileWriteErr     map[string]error
 }
 
+func (f *kanikoFileMock) Getwd() (string, error) {
```

Comment:
Could be avoided when followed [guidance about interfaces](https://github.com/golang/go/wiki/CodeReviewComments#interfaces) in `FileUtils`. But I don't expect you to refactor the code now.. 😉

Reply:
done